provisioning_templates/iPXE/ipxe_intermediate_script.erb

```diff
@@ -6,14 +6,14 @@ snippet: false
 -%>
 #!ipxe
 # Intermediate iPXE script to report MAC address to Foreman
-
 <% (0..32).each do |i| -%>
+
 :net<%= i %>
 isset ${net<%= i -%>/mac} || goto no_nic
 dhcp net<%= i -%> || goto net<%= i+1 %>
 chain <%= foreman_url('iPXE') %>?mac=${net<%= i -%>/mac} || goto net<%= i+1 %>
-<% end -%>
 
```

Comment:
I think this one isn't needed, now we get 2 newlines between each

Reply:
fixed